templates/components/timeToggle.html

```diff
@@ -0,0 +1,14 @@
+{{ define "timeToggle" }}
+  <div class="row border-bottom p-3 mx-0">
+    <div class="col-md-2">Time:</div>
```

Comment:
This breaks the formatting:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/114066135/45d967da-030c-497c-85c6-ca148ccee051)
The time now is placed to much to the left.

Reply:
will take a look for this one. thnx